packages/php-wasm/logger/src/logger.ts

```diff
@@ -62,7 +76,17 @@ export class Logger {
 	 * @param PromiseRejectionEvent event
 	 */
 	private logUnhandledRejection(event: PromiseRejectionEvent) {
-		this.log(`${event.reason.stack}`, 'fatal');
+		this.log(`${event.reason.stack}`, 'Error');
+	}
+
+	public maybeDispatchFatalErrorEvent() {
```

Comment:
Let's inline this method

Reply:
[93e28c0](https://github.com/WordPress/wordpress-playground/pull/1095/commits/93e28c076dcf467518b6d30718311f7cd366562b)